src/main/java/io/xpire/logic/commands/ReplenishCommand.java

```diff
@@ -0,0 +1,84 @@
+package io.xpire.logic.commands;
+
+import static io.xpire.commons.core.Messages.MESSAGE_INVALID_ITEM_DISPLAYED_INDEX;
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+
+import io.xpire.commons.core.index.Index;
+import io.xpire.logic.commands.exceptions.CommandException;
+
+import io.xpire.model.Model;
+import io.xpire.model.item.Item;
+import io.xpire.model.item.Name;
+import io.xpire.model.item.XpireItem;
+import io.xpire.model.item.exceptions.ItemNotFoundException;
+import io.xpire.model.tag.Tag;
+import io.xpire.model.tag.TagComparator;
+
+/**
+ * Adds a {@code ToBuyItem} to the Replenish List.
```

Comment:
Hi sy, did you perhaps mean `ReplenishItem` instead of `ToBuyItem` here?

Reply:
Yeps, I have updated it in my latest commit and will push the changes later on :stuck_out_tongue_closed_eyes: